### PR TITLE
Added further utility documentation to the terraform-concepts.md file

### PR DESCRIPTION
**Pull request by jem3523**
## Overview
Further utility documentation was added to the terraform-concepts.md file so that users can configure multiple profiles, as well as set up a standard function to shorten the log-in process.

## Checklist
- [x ] Proof-reading
- [ no diagrams included] Any diagrams/figures have their source files included

### CODE REVIEW

File: iac/terraform-concepts.md [hunk 1]
```diff
@@ -86,4 +86,59 @@ Once installed, you can run the utility with the below arguments. Your username
 aws-adfs login --adfs-host=ads-fed.northwestern.edu --provider-id urn:amazon:webservices --region us-east-2 --no-sspi
 ```
 
-Once logged in, the utility will prompt you to choose the AWS account to use.
\ No newline at end of file
+Once logged in, the utility will prompt you to choose the AWS account to use.
+
+## More About Configuring Profiles in the Terraform Utility
+
+The Terraform utility acts as a log-in middleware between your TF files and the AWS account into which they will upload when you enter "Terraform Apply". In short, it logs into an AWS account on your behalf during the upload.
```
Comment (cecolby11) [inline iac/terraform-concepts.md:93]: I don't think I'd refer to aws-adfs as a terraform utility -- terraform apply is one example of using it, but the tool just allows us to get temporary AWS credentials for federated login

**Comment by nie7321** [comment]
I merged stable into this cuz it had a weird yarn.lock file and I wasn't sure if it was going to be an issue -- it was not :)

File: iac/terraform-concepts.md (hunk 1 above)
Comment (bab4379) [inline iac/terraform-concepts.md:93]: This is not entirely accurate.  The aws-adfs utility facilitates logging you into AWS and it places your temporary credential information in your .aws/credentials files where many other applications (including terraform) know where to look and how to use it.

**Comment by jem3523** [comment]
Casey,

How do we refer to aws-adfs then?  I understand now that calling it a “terraform utility” is a stretch, but I’m not sure what then call it.

It will be helpful for new users to be able to call it something specific.

-J

File: iac/terraform-concepts.md [hunk 2]
```diff
@@ -86,4 +86,59 @@ Once installed, you can run the utility with the below arguments. Your username
...
+
+A default profile is established based on the first AWS account you use. If you do not want that account to be your default, you can change it. (See below.)
```
Comment (bab4379) [inline iac/terraform-concepts.md:95]: Sorry to be pedantic, but the default profile is established if you do not specify --profile

File: iac/terraform-concepts.md [hunk 3]
```diff
@@ -86,4 +86,59 @@ Once installed, you can run the utility with the below arguments. Your username
...
+{                                                                                     
+	unset AWS_PROFILE
+	aws-adfs login --adfs-host=ads-fed.northwestern.edu --provider-id urn:amazon:webservices --profile $1 --no-sspi
+	export AWS_PROFILE=$1
+}
+```
+
+4. Enter [esc] to exit insert mode
+5. Enter [shift]zz to save and close
+6. If you are using linex, enter:   **source ~/.bash_profile**
```
Comment (bab4379) [inline iac/terraform-concepts.md:141]:
```suggestion
6. If you are using linux, enter:   **source ~/.bash_profile**
```

**Comment by cecolby11** [comment]
plenty of ways to describe it - you could refer to it as a command line tool, or the aws-adfs utility, or a cli tool for aws authentication, or just aws-adfs

**Comment by bab4379** [comment]
This might be more info than needed, but this is the documentation on it from AWS. 
 https://aws.amazon.com/blogs/security/aws-federated-authentication-with-active-directory-federation-services-ad-fs/

I would likely call it a utility that facilitates federated logins to AWS through active directory, but that is a mouthful

**Comment by jem3523** [comment]
How about “AWS federated login utility”?

File: iac/terraform-concepts.md [hunk 4]
```diff
@@ -86,4 +86,58 @@ Once installed, you can run the utility with the below arguments. Your username
 aws-adfs login --adfs-host=ads-fed.northwestern.edu --provider-id urn:amazon:webservices --region us-east-2 --no-sspi
 ```
 
-Once logged in, the utility will prompt you to choose the AWS account to use.
\ No newline at end of file
+Once logged in, the utility will prompt you to choose the AWS account to use.
+
+## More About Configuring Profiles in the Terraform Utility
```
Comment (cecolby11) [inline iac/terraform-concepts.md:91]: Terraform Utility again in the section header line 91. Other than that looks good!

**Pull request merged**